packages/typespec-ts/src/modular/helpers/operationHelpers.ts

```diff
@@ -908,9 +920,11 @@ function serializeRequestValue(
         coreUtilSet.add("uint8ArrayToString");
       }
       return required
-        ? `uint8ArrayToString(${clientValue}, "${format ?? "base64"}")`
+        ? `uint8ArrayToString(${clientValue}, "${
+            getEncodingFormat(type) ?? "base64"
```

Comment:
I think the format does not belong to type here.

Reply:
I don't think I follow? Anyway this change is temporary to unblock the generation of OpenAI generic. This will have to be re-thought with @timovv's work on file upload

Reply:
In parameters, the format is not taged on type, instead it's taged in the parameter itself if I remember correctly. but if it's a model property, the format will be taged on the model property's type.

Reply:
gotcha! Good catch :)